askama_shared/src/generator.rs

```diff
@@ -1106,9 +1129,64 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
                 self.visit_method_call(buf, obj, method, args)?
             }
             Expr::RustMacro(name, args) => self.visit_rust_macro(buf, name, args),
+            Expr::Let(ref variant, ref params, ref val) => self.visit_let(buf, variant, params, val)?,
```

Comment:
```suggestion
            Expr::Let(ref variant, params, ref val) => self.visit_let(buf, variant, params.as_ref(), val)?,
```

(in extension to the last comment)